homeassistant/components/camera/nest.py

```diff
@@ -0,0 +1,93 @@
+"""
+Support for Nest Cameras.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/camera.nest/
+"""
+
+import logging
+from datetime import timedelta
+import requests
+# import voluptuous as vol
+from homeassistant.components.camera import (PLATFORM_SCHEMA, Camera)
+import homeassistant.components.nest as nest
+from homeassistant.util import Throttle
+
+
+DEPENDENCIES = ['nest']
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({})
+# TODO be NestAware subscription aware, (10/min subscribed, 2/min otherwise)
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=6)
+#MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=30)
+
+SIMULATOR_SNAPSHOT_URL = 'https://developer.nest.com/simulator/api/v1/nest/devices/camera/snapshot'
+
+# pylint: disable=unused-argument
```

Comment:
I copy-pasta'd this from another camera implementation, so probably worth a cleanup to remove it anywhere else 😁